### PR TITLE
Add extraEnv to values

### PR DESCRIPTION
**Pull request by pauldn-wttj**
Hello :wave:

Thanks for your work on this Helm chart! We're using it and we'd like to inject extra environment variables, here is a small contribution to be able to do that adding `extraEnv` in the values.

Let me know if there's anything you'd prefer in another way.

Paul

## Tasks:

 - [X] I've made changes
 - [X] I've bumped chart's version in `Chart.yaml`
 - [X] I've added changes to charts `CHANGELOG.md`
 - [X] I've run [`helm-docs`](https://github.com/norwoodj/helm-docs)

### CODE REVIEW

**Comment by MaikuMori** [comment]
Thanks for the PR, this seems reasonable.

Would you mind also bumping Gotenberg version to [8.9.0](https://github.com/gotenberg/gotenberg/releases/tag/v8.9.0) and add changelog? There are no changes in the charts for this version.

Then it's good to merge.

File: charts/gotenberg/values.yaml [hunk 1]
```diff
@@ -94,6 +94,10 @@ volumeMounts: []
 #  - name: tmp-volume
 #    mountPath: /tmp
 
+extraEnv: []
```
Comment (MaikuMori) [inline charts/gotenberg/values.yaml:97]: Maybe add documentation comment `# -- List of extra environment variables for gotenberg container`.

Comment (pauldn-wttj) [reply charts/gotenberg/values.yaml:98]: Version has been bumped as well!

https://github.com/MaikuMori/helm-charts/pull/42/commits/649da8ed6b99e9544819a43d2ca3f68c6c8808ef

**Comment by MaikuMori** [comment]
Thanks, I forgot to add attribution, I'll try to remember to add it next release.

**Comment by pauldn-wttj** [comment]
Thanks a lot for the quick release! :pray:

**Pull request merged**